### DISCUSSION TITLE
Was anyone able to run fastlane on Windows?

### DISCUSSION THREAD

**Discussion by aroyzen1983** [fastlane/fastlane#19664] [Q&A]
We were able to successfully install fastlane on Windows, however anytime we are required to log in to Apple account for example when running `fastlane spaceauth` command we are getting the following error: 
"No such file or directory @ rb_sysopen - /tmp/spaceship1586831345_16204_22439120.log"

Is Mac OS required for that command to run successfully?

**Comment by Yegorich555** [comment]
It doesn't work on Windows 10 in 2025, no idea why, but `fastlane spaceauth` failed with

> No such file or directory @ rb_sysopen - /tmp/spaceship1735813746_28968_6240.log
> Could not login to App Store Connect

At the same time it works fine on MacOS
